src/proc/occlusion-filter.cpp

```diff
@@ -44,20 +67,23 @@ namespace librealsense
        auto height_out = width;
 
        auto out = dest[0];
-       byte buffer[ROTATION_BUFFER_SIZE][ROTATION_BUFFER_SIZE * SIZE];
-       for (int i = 0; i < height; i = i + ROTATION_BUFFER_SIZE)
+       auto buffer_size = maxDivisorRange(height, width, 1, ROTATION_BUFFER_SIZE); 
+      
+       byte *buffer = new byte[buffer_size * buffer_size * SIZE];  
```

Comment:
I can't see delete[]...
Here managed (RAII) memory is preferred, so please switch to std::vector